File: src/main/java/de/maggiwuerze/xdccloader/events/channel/server/ChannelDoneEvent.java
package de.maggiwuerze.xdccloader.events.channel.server;

import de.maggiwuerze.xdccloader.events.EntityDoneEvent;
import de.maggiwuerze.xdccloader.model.entity.Channel;

public class ChannelDoneEvent extends EntityDoneEvent<Channel> {

	public ChannelDoneEvent(Object source, Channel channel) {
		super(source, channel);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/channel/server/ChannelDoneEvent.kt
package de.maggiwuerze.xdccwebloader.events.channel.server

import de.maggiwuerze.xdccwebloader.events.EntityDoneEvent
import de.maggiwuerze.xdccwebloader.model.entity.Channel

class ChannelDoneEvent(source: Any, channel: Channel) : EntityDoneEvent<Channel>(source, channel)